modules/installation-osp-kuryr-settings-active.adoc

```diff
@@ -39,7 +39,7 @@ spec:
       poolBatchPorts: 3 <3>
       poolMaxPorts: 5 <4>
 ----
-<1> Set `enablePortPoolsPrepopulation` to `true` to make Kuryr create new Neutron ports after a namespace is created or a new node is added to the cluster. This setting raises the Neutron ports quota but can reduce the time that is required to spawn pods. The default value is `false`.
+<1> Set `enablePortPoolsPrepopulation` to `true` to make Kuryr create new Neutron ports when the first pod that is configured to use the dedicated network for pods is created in a namespace. This setting raises the Neutron ports quota but can reduce the time that is required to spawn pods. The default value is `false`.
```

Comment:
~new~

Also, sometimes you call them both "ports" and "Neutron ports." Does the extra info make sense in only some contexts?

Reply:
^^^ Yes. Not all ports in ShiftStack docs are Neutron-related.

Reply:
```suggestion
<1> Set `enablePortPoolsPrepopulation` to `true` to make Kuryr create Neutron ports when the first pod that is configured to use the dedicated network for pods is created in a namespace. This setting raises the Neutron ports quota but can reduce the time that is required to spawn pods. The default value is `false`.
```